server/app/views/questiontypes/EnumeratorQuestionFragment.html

```diff
@@ -95,9 +95,9 @@
     <button
       type="button"
       class="cf-enumerator-delete-button usa-button--unstyled"
+      th:data-confirmation-message="#{dialog.confirmDeleteAllButtonsSave(${enumeratorQuestion.getEntityType()})}"
```

Comment:
(question) How does this get trigger?

Reply:
The static javascript is the same across the old and northstar views - so the removeEnumeratorField and removeExistingEnumeratorField handlers are attached in both paths.